tiledb/filestore.py

```diff
@@ -254,33 +254,6 @@ def copy_to(filestore_array_uri: str, file_uri: str, ctx: "Ctx" = None) -> None:
 
         lt.Filestore._uri_export(lt.Context(ctx, False), filestore_array_uri, file_uri)
 
-    def uri_import(self, uri: str, mime_type: str = "AUTODETECT") -> None:
```

Comment:
Please keep the function, mark it deprecated, and throw an error with a message suggesting what to use instead.